source/index.html.md

```diff
@@ -3,13 +3,9 @@ title: API Reference
 
 language_tabs: # must be one of https://git.io/vQNgJ
   - shell
-  - ruby
```

Comment:
No Elixir?

Reply:
lol, we can add Elixir. but then we need to give examples of how they can make requests to our api in Elixir :wow:. Right now we've only catered for shell.

Reply:
Makes the sense :wow: